app/src/main/java/com/animal_villa/dao/IStorageDAO.kt

```diff
@@ -1,4 +1,6 @@
 package com.animal_villa.dao
-
-class IStorageDAO {
+/**
+ * Interface for RetrofitClientInstance implementation
+ */
+interface IStorageDAO {
```

Comment:
Using Interface over classes for DAOs to follow Kotlin's principles of reusability of code. Helps reduce technical debt. https://stackoverflow.com/questions/23561939/why-use-interface-in-dao-design-pattern-or-other-design-patterns